packages/functional-tests/playwright.config.ts

```diff
@@ -11,7 +11,7 @@ const CI = !!process.env.CI;
 // see .vscode/launch.json
 const DEBUG = !!process.env.DEBUG;
 const SLOWMO = parseInt(process.env.PLAYWRIGHT_SLOWMO || '0');
-const NUM_WORKERS = parseInt(process.env.PLAYWRIGHT_WORKERS || '16');
+const NUM_WORKERS = parseInt(process.env.PLAYWRIGHT_WORKERS || '4');
```

Comment:
Just making sure, was this intentional or just something for local test / debug?